Dockerfile.ubuntu

```diff
@@ -0,0 +1,71 @@
+FROM ubuntu:18.04 AS build
+
+RUN apt-get update \
+    && apt-get install -y libssl-dev \
+    llvm-dev clang libclang-dev \
+    wget git cmake build-essential pkg-config
+
+# Rust setup
+ARG RUST_VERSION=1.43.0
+
+ENV RUSTUP_HOME=/usr/local/rustup \
+    CARGO_HOME=/usr/local/cargo \
+    PATH=/usr/local/cargo/bin:$PATH
+
+RUN set -eux; \
```

Comment:
It looks like this step was pulled from the official Rust dockerfile. I don’t think it needs to be part of this PR, but a common maintenance task will be updating the Rust version. Can we document how to do that in a markdown file in the repo root?